features/shared-storage-non-cookie.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
 name: Extending Storage Access API (SAA) to non-cookie storage
```

Not much ink has been spilt on this yet. All I've got is this: https://developers.google.com/privacy-sandbox/blog/saa-non-cookie-storage

```suggestion
name: Non-cookie storage access
```

(rename the file to match)

---

omg so much better